ceph-volume-ansible-prs/build/teardown

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# There has to be a better way to do this than this script which just looks
+# for every Vagrantfile in scenarios and then just destroys whatever is left.
+
+cd $WORKSPACE/tests
```

Comment:
this doesn't matter really because it uses `find`, but I fixed this (and the rest) regardless